executor/client/client.go

```diff
@@ -57,16 +57,20 @@ func (c *Client) GetServiceForFunction(metadata *metav1.ObjectMeta) (string, err
 
 	resp, err := http.Post(executorUrl, "application/json", bytes.NewReader(body))
 	if err != nil {
+		log.Printf("Returning from http post, err : %v", err)
```

Comment:
We've tried to follow a rule of having no printfs in clients.  One reason is that the client may be called from a cli context where you're not allowed to log.  Another reason is that this is a library and you want all this information to be available to the caller, not just the person reading the logs.  If you think these two errors are important enough to distinguish, use `errors.Wrap` or new error types.